.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "src/Combo"]
 	path = src/Combo
 	url = https://github.com/Alexander-Belyi/Combo
-	branch = v2.1
+	branch = master
```

Comment:
Hey @Alexander-Belyi , all looks good, but do we really need to switch to master? All my experience tells me it is safer to pin to something  more fixed

Reply:
Branch is not used to 'pin'. It only shows which branch of submodule to use when we do `git submodule update --remote`.

Reply:
that's exactly my point. I'd rather for each version of the published package to have an explicit pin to the tag (which corresponds to a specific commit hash) rather than an abstract branch. We'll have to update those on new versions, but in the repository, we could still exactly see which version of my combo corresponds to which version of the combo.

Shall we have a short call to sort it out? Perhaps I am missing something

Reply:
Sure, let's have a call. Today or tomorrow is fine.

Reply:
sorry, couldn't find time to react. 
I am merging the PR now, don't want to block it, but let's check how it will work out and rebase on that later